data/routing.txt

```diff
@@ -15,8 +14,15 @@ Bitcoin         | Coinbase   | 1% + 15¢     | On request |
 
 Route           | Provider   | Fee          | When       | Details
 ----------------+------------+--------------+------------+-----------
-Bank Account    | Balanced   | 0            | At payday  | U.S. only
```

Comment:
> What do you mean by "Credit Card belongs to Bank"?

You don't own the Credit Card the same way you own the cash. Credit Card pool of resources is located in Bank, so "belongs" here means that Bank decides what to do with them - if Bank is bankrupt, blocked or something - you have no control over it.